nixos/modules/installer/tools/nixos-generate-config/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "nixos-generate-config"
```

Comment:
I think we should have a single crate (and binary) for the Rust NixOS tools. That should cut down on the binary size if we rewrite nixos-rebuild, nixos-install, switch-to-configuration etc. in Rust.

Reply:
I thought about the same. A multi-call binary could be added in the future to cover that.
I would leave the tools separate though and have multiple crates that may share common utility crates.